### PR TITLE
Fix timeout in sql_fuzzy test

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Fix timeout in `sql_fuzzy` test. Timeout of 600 seconds happens when running `SELECT arrayResize([['\0']], -72295807);` under Thread Sanitizer. But it's not a bug - the query just takes long time to finish.

### CODE REVIEW

File: src/Functions/GatherUtils/Algorithms.h [hunk 1]
```diff
@@ -12,11 +12,15 @@
 namespace DB::ErrorCodes
 {
     extern const int LOGICAL_ERROR;
+    extern const int TOO_LARGE_ARRAY_SIZE;
```
Comment (alexey-milovidov) [inline src/Functions/GatherUtils/Algorithms.h:15]: These changes are irrelevant.

File: tests/queries/0_stateless/00746_sql_fuzzy.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ $CLICKHOUSE_CLIENT -q "select name from system.table_functions format TSV;" > "$
 # if you want long run use: env SQL_FUZZY_RUNS=100000 clickhouse-test sql_fuzzy
 
 for SQL_FUZZY_RUN in $(seq "${SQL_FUZZY_RUNS:=10}"); do
-    env SQL_FUZZY_RUN="$SQL_FUZZY_RUN" "$CURDIR"/00746_sql_fuzzy.pl | $CLICKHOUSE_CLIENT --format Null --max_execution_time 10 -n --ignore-error >/dev/null 2>&1
+    env SQL_FUZZY_RUN="$SQL_FUZZY_RUN" "$CURDIR"/00746_sql_fuzzy.pl | timeout 60 $CLICKHOUSE_CLIENT --format Null --max_execution_time 10 -n --ignore-error >/dev/null 2>&1
```
Comment (alexey-milovidov) [inline tests/queries/0_stateless/00746_sql_fuzzy.sh:16]: This is the only significant change.

**Review by alexey-milovidov — COMMENTED** [review]
No additional tests will be added.

**Review by KochetovNicolai — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
Ok but could you please clarify failed tests?

**Comment by KochetovNicolai** [comment]
`test_materialize_mysql` was broken in master.
Fixed in https://github.com/ClickHouse/ClickHouse/commit/76df13c31c2a1165694e969d21bac4e2cdc3cbfe

**Pull request merged**